app/controllers/idv/in_person_controller.rb

```diff
@@ -7,7 +7,7 @@ class InPersonController < ApplicationController
 
     FSM_SETTINGS = {
       step_url: :idv_in_person_step_url,
-      final_url: :account_url,
+      final_url: :idv_review_url,
```

Comment:
> This ends up routing to the phone page which is the correct next page in the flow. This is adopted from the doc auth flow

I find it a bit confusing to direct them to the review step and expect that to be redirected to the phone step (rather than just directing them straight to the phone step), but I'm not too bothered by it if it's just copying from the doc_auth flow. Alternatively, we could update both to go to the phone step. I'm not aware of any reason a user would be able to skip the phone step.

Reply:
I'll update it to go directly to the phone page